#Request
Header header
bool laneChangeActive

---
#Response
bool adjusted